integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -128,16 +128,16 @@ spec:
               name: {{ $val | lower }}-{{ $key | replace "_" "" }}
                           {{- end}}
                         {{- end}}
-                      {{- end}}
                       {{- else}}
-                      {{- if eq .type "hostPath"}}
+                        {{- if eq .type "hostPath"}}
             - name: {{ .alias | lower }}
               mountPath: {{ .path }}
-                      {{- else }}
+                        {{- else }}
             # report error and exit
             {{ .name }} with {{ .type }} is not supported in shortCircuitPolicy local
-                      {{- end }} 
-                    {{- end }} 
+                        {{- end }} 
+                      {{- end }}
+                    {{- end}} 
```

Comment:
I think we can better reduce the number of nested levels by using `{{- with .Values.tieredstore.level }}{{ end -}}` instead of `{{if .Values.tieredstore}}{{if .Values.tieredstore.level}}{{end}}{{end}}`. 

Example:
```
            {{- with .Values.tieredstore.level }}
                {{- range . }}
                  {{- if .mediumtype }} ... {{end}}
               {{- end }}
            {{- end }}
```